addons/web/static/src/views/fields/url/url_field.js

```diff
@@ -43,7 +43,7 @@ export const urlField = {
             help: _t("If True, the url will be used as it is, without any prefix added to it."),
         },
     ],
-    supportedTypes: ["char"],
+    supportedTypes: ["char", "text"],
```

Comment:
There is no logic to support type "text" for the URL, URL's shouldn't have line breaks.
I thisnk, you should modify the field to have the correct type.